testing/test_transforms.py

```diff
@@ -0,0 +1,83 @@
+import unittest
+import GPflow
+import GPflowOpt
+import tensorflow as tf
+from GPflow import settings
+import numpy as np
+
+float_type = settings.dtypes.float_type
+np_float_type = np.float32 if float_type is tf.float32 else np.float64
+
+
+class DummyTransform(GPflowOpt.transforms.DataTransform):
+
+    def __init__(self, c):
+        self.value = c
```

Comment:
not sure if dummytransform is the correct name if it actually multiplies by a constant